pkg/client/unversioned/request_test.go

```diff
@@ -729,6 +734,39 @@ func TestDoRequestNewWay(t *testing.T) {
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &reqBody)
 }
 
+// This test assumes that the client implementation backs off exponentially, for an individual request.
+func TestBackoffLifecycle(t *testing.T) {
+	count := 0
+	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		count++
+		t.Logf("attempt %d", count)
+		//requests "1" and "2" both fail.
```

Comment:
The comment is "right"  but... if its not useful i can get rid of it (both requests below should fail with 500s).